doc/Type/Sequence.pod6

```diff
@@ -7,8 +7,13 @@
     role Sequence does PositionalBindFailover { }
 
 This role implements a series of methods for converting sequences and
-accessing their elements. These methods have in common that they all
-access the underlying iterator if the sequence has not been
+accessing their elements. They should work on all sequences, regardless
```

Comment:
If "should work" refers to "series of methods" I think that that would be an "it", not a "they", but it would probably be better to rephrase it with a relative clause like "which should work on..."

Reply:
The word “they” is correct for “a series of methods”; “series” can be either singular or plural, and in this case it’s plural. And “they” can refer back to “these methods” as well, which is very clearly plural. Switching to singular “it” is confusing to me; I can’t read its bound referent as either “these methods” or “series”.

“They” vs a relative clause is a stylistic matter, but if you want the predicate to refer to “series of methods” rather than “these methods”, it would have to move to an unnatural place in the passage. “They” pretty clearly can encompass both (and encompassing just one or the other doesn’t result in a bad reading).

(Sorry for dipping the linguistics oar in. But bound references in English was one of my research topics.)

Reply:
I read "a series" as singular, so on that aspect I have to agree with JJ. But using "it" doesn't feel natural and using "which" after "a series of methods for converting sequences and accessing their elements" is ambiguous. So I will change "they" back to "these methods".